e2e/tests/send.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Tx Namespace
+
+  Scenario: User can send uPOKT
+    Given the user has the pocketd binary installed
+    When the user sends 10000 uPOKT to another address
+    Then the user should be able to see standard output containing "txhash:"
+    And the user should be able to see standard output containing "code: 0"
+    And the pocketd binary should exit without error
```

Comment:
Couple of changes I'd make to this:
1. Validate which account has 10000 uPOKT. I don't know who the user is
2. Explicitly state WHEN `account a1 sends 10000 to account a2`
3. Verify that the final balance of a1 is lower
4. Verify that the final balance of a2 is higher

Now we have a real regression test.